test/end-to-end/core.sh

```diff
@@ -167,14 +167,17 @@ os::cmd::expect_success 'oc whoami'
 
 echo "[INFO] Running a CLI command in a container using the service account"
 os::cmd::expect_success 'oc policy add-role-to-user view -z default'
-oc run cli-with-token --attach --image=openshift/origin:${TAG} --restart=Never -- cli status --loglevel=4 > ${LOG_DIR}/cli-with-token.log 2>&1
+#TODO(skuznets): remove sleeps after k8s attach becomes more synchronized
+oc run cli-with-token --attach --image=openshift/origin:${TAG} --restart=Never -- sleep 10 && cli status --loglevel=4 > ${LOG_DIR}/cli-with-token.log 2>&1
```

Comment:
Oh, yuck. Right. So we want 
```
$ oc run cli-with-token --attach --image=openshift/origin:${TAG} --restart=Never --command 'sleep 10; cli status' ......
```

@Miciah unclear why this is `cli status` and not `oc status`... but that's not a change I made